homework-9/script-1.js

```diff
@@ -0,0 +1,187 @@
+/*Створити функцію-конструктор Людини. Властивості, які будуть описувати екземпляр Людини - ім'я, вік, стать, 
+національність, країна, список країн для подорожей. Створити універсальні функції - знайомство, прокидатися, 
+засинати й список країн, які Людина хоче відвідати. Роботу цих функцій можна виводити через console.log.
+ Але в кожній з функцій повинно використовуватись якомога більше характеристик 
+конкретної Людини на якій ця функція викликається (підказка: тут треба використати call/apply)*/
+function Person(namePerson, agePerson, sexPerson, nationalityPerson, countryPerson, listCountryForTravelling) {
```

Comment:
It's better to give names backward  
```suggestion
function Person(personName, personAge, personSex, persoNationality, personCountry, listCountryForTravelling) {
```